src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/ProjectDescription/IDependencyGraphService.cs

```diff
@@ -0,0 +1,9 @@
+using NuGet.ProjectModel;
+
+namespace Microsoft.DotNet.MSIdentity.Project
+{
+    internal interface IDependencyGraphService
```

Comment:
>IDependencyGraphService [](http://example.com/codeflow?start=23&length=23)

comments

Reply:
fixed and moved https://github.com/dotnet/Scaffolding/pull/1593

Reply:
fixed and moved https://github.com/dotnet/Scaffolding/pull/1593